packages/docs/src/locales/en/en.json

```diff
@@ -758,7 +758,8 @@
       },
       "events": {
         "clear": "Emitted if select value has been cleared",
-        "updateSearch": "Emitted if search value has changed"
```

Comment:
When I press UP button - cursor goes directly to "five" option.
It should not ignore the option in-between even though text value is the same (id is not!)

![image](https://user-images.githubusercontent.com/5394573/145559586-ac7ff354-5206-49ec-9eb3-7d1731a95a3f.png)

Reply:
Let's move fixing these to another PR

Reply:
fixed it:
![chrome-capture](https://user-images.githubusercontent.com/55198465/145571057-851301e8-5c71-4415-820b-afe8c3ca08fa.gif)